profiles/utilities.py

```diff
@@ -22,3 +26,21 @@ def guess_index_name(name: str) -> str:
 
 def remove_none_values(items: dict) -> dict:
     return dict(filter(lambda item: item[1] is not None, items.items()))
+
+
+def validate_json(data: dict, schema_name: str):
+    schema_dir = 'test/schema'  # Will be replaced with elife_api.schema_directory
```

Comment:
Maybe make this an explicit TODO?

Reply:
In fact, we can load them now can't we?

Reply:
Yes was going to wait until the `api-validator` was complete but no real reason not to use it for the schemas now. I have included it as a a dependancy now and amended the temporary validation function for the time being.